lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3218,6 +3229,13 @@ namespace {
             return nullptr;
           }
         }
+        // Use the exit function from _SwiftConcurrency.h as it is platform
+        // agnostic.
+        if ((topLevelModuleEq(decl, "Darwin") ||
+             topLevelModuleEq(decl, "SwiftGlibc")) &&
+            decl->getDeclName().isIdentifier() && decl->getName() == "exit") {
+          return nullptr;
+        }
```

Comment:
@zoecarver: is there some reason  for not doing this check like `!topLevelModuleEq(decl, "SwiftConcurrencyShims") && decl->getDeclName().isIdentifier() && decl->getName() == "exit"` instead? Hardcoding `SwiftConcurrencyShims` is bad, but at least it is just one module name, and not two. I found out about this because `SwiftGlibc` is the name of the overlaid module provided by Swift, but what if the system provides its own modulemap for the system headers and it is called differently?

Reply:
We want to import user-defined overloaded or nested exit functions.